includes/rrdtool.inc.php

```diff
@@ -103,6 +103,10 @@ function rrdtool_graph($graph_file, $options)
     /** @var Proc $rrd_sync_process */
 
     if (rrdtool_initialize(false)) {
+        $now_timestamp = date('Y-m-d H:i:s', time());  // timestamp
```

Comment:
You don't need the second argument here.
Also, this probably isn't the best place for this, since it adds it to EVERY command.  It should probably be in the rrd_graph command or just graph.inc.php

Reply:
removed redundant argument. next step will be webui option